scripts/one-trust-geo-script.js

```diff
@@ -1,15 +1,50 @@
+import { addMartechStack } from './delayed.js';
+
 const createOptanonWrapper = () => {
   const script = document.createElement('script');
   script.type = 'text/javascript';
   script.textContent = 'function OptanonWrapper() { }';
   document.head.append(script);
 };
 
+const createSatTrack = () => {
+  const script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.innerHTML = `const getCookie = (sName) => {
+    const sValue = "; " + document.cookie;
+    const aParts = sValue.split("; " + sName + "=");
+    if (aParts.length === 2) return aParts.pop().split(";").shift();
+  };
+
+  const satTrackHandler = () => {
+    const sSatTrackCookie = getCookie('sat_track');
+
+    if (typeof sSatTrackCookie !== 'undefined') {
+      return;
+    }
+
+    // list of geos where only strictly necessary cookies are allowed
+    const aSNGeos = ["GB", "BE", "IE", "FR", "RU", "ES", "IT", "DE", "AT", "CH"];
+    if (aSNGeos.indexOf(window.otUserLocation) !== -1) {
+      document.cookie = "sat_track=false;path=/;secure;max-age=31536000";
```

Comment:
are we sure about the max age values?

Reply:
As per analytics team, that's the correct value.